pytorch_lightning/trainer/connectors/checkpoint_connector.py

```diff
@@ -79,8 +78,17 @@ def restore(self, checkpoint_path: str, on_gpu: bool):
         - schedulers
         - optimizer
         In detail, check return value description of `dump_checkpoint`
+
+        Returns:
+           `True` if restored successfully else `False`
         """
 
+        # Try to read the checkpoint file at `checkpoint_path`. If not exist, do not restore checkpoint.
+        fs = get_filesystem(checkpoint_path)
+        if not fs.exists(checkpoint_path):
+            rank_zero_warn("No checkpoint file exists at `resume_from_checkpoint`. Start from scratch")
+            return False
+
         # if on_gpu:
```

Comment:
Why are those comments there ?

Reply:
This warn exists as a result of this discussion > https://github.com/PyTorchLightning/pytorch-lightning/pull/4402#issuecomment-723771940  
There is no strong reason why this warn is in `restore`.  
Warns could be in another place (checking bool return value of `restore` enable warn switching.)